<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>All routes</string>
    </property>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>711</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deletebtn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>230</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Route</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="createnewroutebox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Create new route</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>671</width>
       <height>192</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Route ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Starting Point</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Destination</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>R-1002</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>gulshan</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>clifton</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>9:40 AM</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>Inactive</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>R-1059</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Malir</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>Korangi</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>1:05 PM</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>Active</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="createRoutebtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New Route</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cstartingPtlinededit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cdestinationLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cend_timelineedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cStartTimelineedit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Starting_point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start_time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End_Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="updateRoutebox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>321</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Create new route</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>671</width>
        <height>192</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Route ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Starting Point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Destination</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>R-1002</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>gulshan</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>clifton</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>9:40 AM</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>Inactive</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>R-1059</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>Malir</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>Korangi</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>1:05 PM</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>Active</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create New Route</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting_point:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Start_time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>End_Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination:</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="cStatuscombobox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="updateroutegrpbox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Update route</string>
    </property>
    <widget class="QPushButton" name="updateRoutebtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Route</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urouteIDlineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ustartPtlineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uEnd_timelineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uStart_timelineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uDestinationlineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Starting_point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start_time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End_Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Route_ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uStatusComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="dashboardbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
